app/models/agents/rss_agent.rb

```diff
@@ -53,18 +52,43 @@ def default_options
       Events look like:
 
           {
+            "feed": {
```

Comment:
Actually there was a note at the bottom but it was probably unnoticeable.  I've added a couple of additional notes about some fields, so it has hopefully gained its presence.